css/responsive.css

```diff
@@ -11,3 +11,29 @@ only screen and (max-width: 800px),
     width: 97vw !important;
   }
 }
+
+@media
+only screen and (max-width: 640px){
+	.verticle-list-in-rd li a .fa-stack{
```

Comment:
@Huyeng where is this class being used? Shouldn't it be called **vertical-list-in-rd**?